src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] salaries = new int[names.length];
+        String[] stringDateFromArray = dateFrom.split("\\.");
+        String[] stringDateToArray = dateTo.split("\\.");
```

Comment:
let's solve this task without arrays, that will make code simplier

Reply:
done